### PR TITLE
Improve Pull request template

### PR DESCRIPTION
**Pull request by Andre601**
Thank you for contributing! Please check the following things before submiting your PR:

Required:
- [x] I have read and followed the [contributing guidelines](https://github.com/Readme-Workflows/recent-activity/blob/main/.github/CONTRIBUTING.md)

If necessary:
- [ ] I have updated the README and documentation.
- [ ] I have updated the ChangeLog with the changes I have made.

## Description

Improves the Pull request template to be more informative and structured.

Closes #

---------------------------------------------------------------------

**Note:** we will close your PR without comment if you do not check the required boxes above and provide ALL requested information.

### CODE REVIEW

**Review by abhijoshi2k — APPROVED** [review]
Looks good!

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,22 +1,38 @@
...
-- [ ] I have updated the ChangeLog with the changes I have made.
+## Checks
+<!-- Please "check" the below options by replacing [ ] with [x] -->
 
-## Description
+I have...
 
-Replace this line with a description about the changes you have made in this PR e.g. Corrected spelling mistakes in docs
+- [ ] read and understood the [Contributing Guidelines][contributing]
+- [ ] Updated any nessecary files such as the README.md and/or CHANGELOG.md.
```
Comment (PuneetGopinath) [inline .github/PULL_REQUEST_TEMPLATE.md:16]: I think it should be mentioned, that if possible, maintainers can also help them in doing this, that's why I kept them in if nessary section.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,22 +1,38 @@
...
 
-<!--
-If this pull request closes an issue, then add the issue number below
--->
+## Type of Pull request
+<!-- Please "check" the below options by replacing [ ] with [x] -->
+<!-- ONLY select one option! -->
 
-Closes #
+- [ ] **Minor Change**
```
Comment (PuneetGopinath) [inline .github/PULL_REQUEST_TEMPLATE.md:22]: Please also add Bug fix in this list.

**Review by PuneetGopinath — CHANGES_REQUESTED** [review]

**Comment by PuneetGopinath** [comment]
@Andre601 Please resolve the changes I requested.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (Andre601) [reply .github/PULL_REQUEST_TEMPLATE.md:16]: The text itself should already be clear enough with "any necessary files"

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (Andre601) [reply .github/PULL_REQUEST_TEMPLATE.md:22]: Implemented.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (PuneetGopinath) [reply .github/PULL_REQUEST_TEMPLATE.md:16]: Ok

**Review by PuneetGopinath — APPROVED** [review]

**Pull request merged**